### PR TITLE
Use the correct schema suffix when retrieving avro schemas

### PR DESCRIPTION
**Pull request by acmiyaguchi**
This fixes a bug with the SchemaStore where the wrong path is provided when fetching a schema. This adds failing tests and a patch for fixing the tests.

Additionally, this updates the AvroSchemaStore to take advantage of schema aliasing.

### CODE REVIEW

File: ingestion-beam/bin/generate-avro-test-resources.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+if __name__ == "__main__":
+    logging.basicConfig(level=logging.DEBUG)
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--output-path",
+        type=str,
+        default=os.path.join(INGESTION_BEAM_ROOT, "src/test/resources/testdata"),
+    )
+    args = parser.parse_args()
+    main(args.output_path)
```
Comment (jklukas) [inline ingestion-beam/bin/generate-avro-test-resources.py]: Is this file intended to be included in this PR?

File: ingestion-beam/src/main/java/com/mozilla/telemetry/schemas/SchemaStore.java [hunk 2]
```diff
@@ -51,8 +51,9 @@ public T getSchema(Map<String, String> attributes) throws SchemaNotFoundExceptio
       throw new SchemaNotFoundException("No schema for message with null attributeMap");
     }
     // This is the path provided by mozilla-pipeline-schemas
-    final String path = StringSubstitutor.replace("${document_namespace}/${document_type}/"
-        + "${document_type}.${document_version}.schema.json", attributes);
+    final String path = StringSubstitutor.replace(
+        "${document_namespace}/${document_type}/" + "${document_type}.${document_version}",
```
Comment (jklukas) [inline ingestion-beam/src/main/java/com/mozilla/telemetry/schemas/SchemaStore.java]:
```suggestion
        "${document_namespace}/${document_type}/${document_type}.${document_version}",
```

File: ingestion-beam/bin/generate-avro-test-resources.py (hunk 1 above)
Comment (acmiyaguchi) [reply ingestion-beam/bin/generate-avro-test-resources.py]: Yes, it's based on the branch in #521, which includes non-empty schemas.
